nx-X11/programs/Xserver/fb/Imakefile

```diff
@@ -34,7 +34,7 @@ SUBDIRS = module
 #include <Server.tmpl>
 
 #ifdef FbNoPixelAddrCode
-DEFINES=-DFBNOPIXADDR -DFBNO24BIT
```

Comment:
fb/Imakefile:

> Should this really be part of this commit? Looks somewhat misplaced?

Actually, the Imakefile change is wanted in that way. I need to inject NX_DEFINES into DEFINES, so I had to move DEFINES out of the way, introduce FB_DEFINES and later on merge NX_DEFINES and FB_DEFINES into DEFINES again.